### ISSUE TITLE
md file can't use import external resources

### ISSUE THREAD

**Issue by Mamorial** [vuejs/vitepress#922]
### Describe the bug

md file can't use import @vicons/antd

**My Code**
:::demo
```vue
<template>
    <wp-icon name="DeleteOutlined"></wp-icon>
</template>

<script lang="ts" setup>
import {RightOutlined} from '@vicons/antd';
</script>
```
:::

![image](https://user-images.githubusercontent.com/26594629/177461734-af43c1d2-9a16-47a1-adfd-cdcf9dd6956c.png)

**Browser Error**
![image](https://user-images.githubusercontent.com/26594629/177461781-3b0a4605-9833-46a4-a8a3-50caa0cd1bb6.png)

### Reproduction

md file use import

### Expected behavior

md file can use import node_modules package

### System Info

```shell
"vitepress": "1.0.0-alpha.4"
"@vicons/antd": "^0.11.0"
"vite": "^2.9.0"
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by Mamorial** [comment]
"vitepress": "0.22.4"

**Comment by brc-dd** [comment]
What is `:::demo`?

**Comment by Mamorial** [comment]
:::demo Has it been abandoned? I want to run VUE code in MD file

**Comment by brc-dd** [comment]
I don't think there was any such thing in VitePress. You can use Vue directly in markdown like this:

```md
# Hey there

<right-outlined />

<script setup lang="ts">
import { RightOutlined } from '@vicons/antd'
</script>
```

**Comment by Mamorial** [comment]
yes i know, but I want to render directly in MD files, like the following
![image](https://user-images.githubusercontent.com/26594629/177484334-90230efe-485e-4c42-8053-e40e5755ccb1.png)

**Comment by brc-dd** [comment]
Yeah, that's what I'm saying, you don't need to put that demo block. Put that content directly inside your md file:

![image](https://user-images.githubusercontent.com/40380293/177484574-ade9dbff-5221-4fa2-b30c-28241efbeb38.png)

**Comment by Mamorial** [comment]
I used this theme
![image](https://user-images.githubusercontent.com/26594629/177484498-ffd3514c-acb6-4268-9a6f-b7d5dfbf5c9d.png)

**Comment by brc-dd** [comment]
That theme is not official. You can create issues on its repo if something isn't working. But this clearly isn't an issue with VitePress.

**Comment by brc-dd** [comment]
I had wrote a simple component that did that demo thing for an issue here. But I'm unable to find that. As for the element plus, you can do this: https://github.com/vuejs/vitepress/issues/603#issuecomment-1103884818. If you can write a demo plugin for yourself then you don't need that theme at all IG. I am closing this for now. If you need some help then create a discussion or ask on Vue's discord (https://chat.vuejs.org/).
